packages/localize/src/modes/transform.ts

```diff
@@ -39,6 +39,10 @@ export interface TransformOutputConfig {
   localeCodesModule?: string;
 }
 
+type TypeScriptTransformerFactoryFactory = (
```

Comment:
if a bean gets involved we've gone too far

Reply:
I couldn't resist this name